### DISCUSSION TITLE
Issues Overriding the Tethys portal app registration page

### DISCUSSION THREAD

**Discussion by Curtis-96** [tethysplatform/tethys#1005] [General]
I am trying to override the app registration page at /oauth2_provider/applications/register and have successfully replaced the template for the page with my own. The problem is I am not seeing any of the fields from the form variable in the template that I defined in a separate controller. Is there a way to render this new form in the Tethys portal app registration page template?

**Comment by sdc50** [comment]
@Curtis-96 I know that this was resolved, but I don't remember exactly what the issue was. Was it that we weren't overriding the controller from `django-oauth-toolkit` correctly?
